uint16 m_signature
uint16 m_x
uint16 m_y
uint16 m_width
uint16 m_height
uint16 m_angle
uint16 m_index
uint16 m_age
